app/controllers/targets_controller.rb

```diff
@@ -0,0 +1,13 @@
+class TargetsController < ApplicationController
+  before_action :authenticate_user!
```

Comment:
In the future, you will need to authenticate the user in more than one controller. Create an ApiControllerUser and inherit from it.